content/warnings/dont-call-proptypes.md

```diff
@@ -4,47 +4,48 @@ layout: single
 permalink: warnings/dont-call-proptypes.html
 ---
 
-> Note:
+> Uwaga:
```

Comment:
Ten plik nie powinien się tu znaleźć. Wygląda na to, że nie zaktualizowałeś brancha przed stworzeniem kolejnego. Wyrzuć go, bo nie chcę mi się go drugi raz sprawdzać ;)